turbinia/workers/file_system_timeline.py

```diff
@@ -81,9 +82,9 @@ def run(self, evidence, result):
       status = 'Generated file system timeline containing [{0:d}] entries'.format(
           number_of_entries)
       result.close(self, success=True, status=status)
-    except dfvfs_errors.ScannerError as exception:
-      result.log('Error generating bodyfile {0!s}'.format(exception))
+    except (dfvfs_errors.ScannerError, dfvfs_errors.BackEndError) as exception:
```

Comment:
So AFAIK this will catch the exception but still not generate the timeline right? Seeing the error come up a few times so little worried on how prevalent it may be esp if we opt out Plaso for this in Triage recipes.

I'm wondering if that error is related to a single entry/inode it couldn't parse or if issue with whole partition

Reply:
My understanding of the error you mentioned (see #1001) is that dfvfs.lib.errors.BackEndError indicates dfvfs failed to read the volume (and this may be because the source file was not found or not a valid volume), so a timeline would not have been generated, which is the correct behavior for that error.  @joachimmetz is this correct?

Reply:
Oh okay that actually makes more sense as well

May just have to check the source data itself for any issues

Reply:
yes in this case a timeline will not be generated since the file system cannot be read, seeing the backtrace in https://github.com/google/turbinia/issues/1001#issuecomment-1035893347 can you point me to the location of the image (out of band) it looks like a format edge case of the ext file system not previously observed

Reply:
Also this block can raise several other exceptions, such as open() raising IOError.

To address @wajihyassine concern the code could be rewritten to use next() explicitly and catch some exceptions for individual file entries

Reply:
I made some changes to be able to handle possible exceptions per file entry. 

Note that this will need to change (see #983) once the new version of dfImagetools is released. Until then, i think this should address the concern.